backend/ethereum/channel/funder.go

```diff
@@ -59,6 +62,12 @@ type Funder struct {
 var _ channel.Funder = (*Funder)(nil)
 
 // NewFunder creates a new ethereum funder.
+//
+// In addition to the `Fund` method required by the `Funder` interface, it also
```

Comment:
I think this comment is not necessary and hard to maintain. Not necessary because the doc will also list the functions. And hard to maintain because if we change the function, we need to change this comment which is easy to miss.